Assets/Scripts/InterpolateTransform.cs

```diff
@@ -21,7 +21,7 @@ public enum InterpolateTransformTriggerMethod
     BOTH
 }
 
-public class InterpolateTransform : MonoBehaviour
+public class InterpolateTransform : InteractableAbstract
```

Comment:
This is meant to be a generic motion interpolation script. It shouldn't be extending InteractableAbstract as it will be used in many different ways (Interaction being one of them). In case of this being used via interaction, we can handle this separately. Revert this back to it's original state